lib/presentation/screen/user_profile/user_profile_screen.dart

```diff
@@ -352,7 +352,7 @@ class _UserProfileScreenState extends State<UserProfileScreen> with TickerProvid
                                                           onPressed: () {
                                                             if (snapshot.hasData) {
                                                               if (snapshot.data!.status == AmityFollowStatus.BLOCKED) {
-                                                                amityUser.relationship().unblock().then((value) {
+                                                                amityUser.unblockUser().then((value) {
```

Comment:
Does it should be amityUser.relationship().unblockUser()?

Reply:
@bnaing Its direct extension method of Amity User Object.